.travis.yml

```diff
@@ -21,6 +21,7 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
+  - if [[ "$WITH_COVERAGE" != "true" ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
Will do. Merging while mobile so I'm going to go ahead and pull this in and edit after the fact.